device-types/Cisco/WS-C6509-E.yaml

```diff
@@ -32,6 +25,6 @@ module-bays:
   - name: Slot 9
     position: '9'
   - name: PS1
-    position: '1'
```

Comment:
The issue with changing the module bay position, is that the module variable in the module template will only be expecting a number rather than PS1.